dot_env

```diff
@@ -0,0 +1,6 @@
+REGRESSION_TEST_OUTPUT_BUCKET=<some bucket like harmony-sit-regression-tests>
```

Comment:
Also, since it's not used except in the deployed environment (i.e. not sourced by run_notebooks.sh) I think moving it to the script/ folder may solve the qualms here